packages/affine/shared/src/utils/insert.ts

```diff
@@ -1,3 +1,5 @@
+import { generateKeyBetween } from 'fractional-indexing';
```

Comment:
please copy this https://github.com/toeverything/AFFiNE/blob/59dfc80bc2019e87a13cdbdf8bd466db32c02e53/packages/common/infra/src/utils/fractional-indexing.ts#L11
 from affine

much safer then origin fractional-indexing